static/src/stylesheets/module/facia/_slices.scss

```diff
@@ -239,6 +210,29 @@ Hence why a greater depth of selector specificity is needed.
     }
 }
 
+.fc-slice__popular-mpu {
+    .ad-slot {
+        min-height: 274px;
```

Comment:
That's a good point. Same for the `286px` a few lines below. I'll try and see what we can do here.